channels/chan_usrp.c

```diff
@@ -593,6 +595,7 @@ static int usrp_xwrite(struct ast_channel *ast, struct ast_frame *frame)
 				remque((struct qelem *) qp);
 				ast_free(qp);
 			}
+			ast_log(LOG_WARNING, "Channel %s: Receive queue exceeds the threshold of %lu\n", ast_channel_name(ast), USRP_VOICE_FRAME_SIZE);
```

Comment:
Does it help knowing anything about how much we are exceeding the threshold ?

Reply:
Not really, this should never happen.  Sending UDP packets does not block.

If this occurs something bad has happened.